src/include/cache.h

```diff
@@ -72,6 +72,7 @@ struct __wt_cache {
     uint64_t bytes_dirty_total;
     uint64_t bytes_evict;      /* Bytes/pages discarded by eviction */
     uint64_t bytes_hs;         /* History store bytes inmem */
+    uint64_t bytes_hs_dirty;   /* History store bytes inmem dirty */
```

Comment:
I'd like to flag that these aren't necessarily current in the comment here (`bytes_hs` and `bytes_hs_dirty`). Possibly add the words "Cached and potentially outdated" as a prefix.

Reply:
I have added a comment for the two variables belonging to history store.